src/main/kotlin/com/autonomousapps/extension/IssueHandler.kt

```diff
@@ -23,15 +25,17 @@ import javax.inject.Inject
  */
 open class IssueHandler @Inject constructor(objects: ObjectFactory) {
 
+  private val undefined = objects.newInstance<Issue>()
```

Comment:
Semantically this is a constant, but because we need the object factory to instantiate it, shouldn't use SHOUT_CASE.